### PR TITLE
Translated the README.md to Arabic Language 

### LINKED ISSUE #3: Translate the README file into an Arabic language

### PR DESCRIPTION
**Pull request by idrismusa4**
Translate README.md file to Arabic as state in issue #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,82 +4,82 @@
...
 
-# brainfuc*k interpreter: a simple brainfuc*k interpreter and REPL writen in rust 🦀
+# ⁭مترجم brainfu* k: مترجم بسيط من Brainfuc * K و REPL مكتوب في Rust 🦀
 
-## Features
-- Run brainfuc*k code from a file 💫
-- Run brainfuc*k live code from the REPL 🚀
-- Support utf-8 characters including emojis 😍 (optional)
-- You can control in the array size 📏
+## سمات
```
Comment (0x61nas) [inline README.md]: The correct translation for `Features` is `مميزات` or `المميزات` not `سمات`

File: README.md [hunk 2]
```diff
@@ -4,82 +4,82 @@
 
 </p>
 
-# brainfuc*k interpreter: a simple brainfuc*k interpreter and REPL writen in rust 🦀
+# ⁭مترجم brainfu* k: مترجم بسيط من Brainfuc * K و REPL مكتوب في Rust 🦀
```
Comment (0x61nas) [inline README.md]: the correct translation for a description is:
مترجم بسيط وREPL  للغة brainfu*k مكتوب باللغة Rust

Comment (0x61nas) [reply README.md]: Or
مكتوب بواسطة لغة Rust

File: README.md [hunk 3]
```diff
@@ -4,82 +4,82 @@
...
 
-## Features
-- Run brainfuc*k code from a file 💫
-- Run brainfuc*k live code from the REPL 🚀
-- Support utf-8 characters including emojis 😍 (optional)
-- You can control in the array size 📏
+## سمات
+- قم بتشغيل كود brainfu * k من ملف 💫
```
Comment (0x61nas) [inline README.md]: Good translation, but we don't need to `قم`sentence

File: README.md [hunk 4]
```diff
@@ -4,82 +4,82 @@
...
+- قم يسمح Brainfu * k الكود المباشر من REPL 🚀
```
Comment (0x61nas) [inline README.md]: we don't need to `قم`sentence

File: README.md [hunk 5]
```diff
@@ -4,82 +4,82 @@
...
+- دعم أحرف UTF-8 بما في ذلك الرموز التعبيرية 😍 (اختياري)
+- يمكنك التحكم في حجم المصفوفة 📏
 
-## Install
+## تثبيت
 
-- from crates.io
+- تثبيت crates.io:
```
Comment (0x61nas) [inline README.md]: the correct translation for `from` is `من` not `تثبيت`

Comment (0x61nas) [reply README.md]: You can translate it to `تثبيت من` also

**Review by 0x61nas — CHANGES_REQUESTED** [review]
The translation is not accurate and needs some correction, also why did you replace the original file?
You are supposed to create a new file containing the translation
`README-ar.md`

File: README.md (hunk 5 above)
Comment (idrismusa4) [reply README.md]: Okay will do

**Comment by idrismusa4** [comment]
Okay I will create a new file and add a new pull request if it's okay?

**Comment by 0x61nas** [comment]
You don't need to create a new PR, leave this open and any new commit will reach it

**Comment by 0x61nas** [comment]
You can rename the file you modified to README-ar.md and do not forget to return the original file

**Comment by 0x61nas** [comment]
And don't forget to put a link to the translation version in the original README file, you can imitate this: https://github.com/anas-elgarhy/cmus-rpc-rs/pull/17

**Comment by idrismusa4** [comment]
> You can rename the file you modified to README-ar.md and do not forget to return the original file

Okay will do

**Comment by idrismusa4** [comment]
> And don't forget to put a link to the translation version in the original README file, you can imitate this: https://github.com/anas-elgarhy/cmus-rpc-rs/pull/17

Okay thanks for the assistance

File: README-ar.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+## لكى يفعل
+- [ ] أضف المزيد من الاختبارات
+- [ ] أضف المزيد من الأمثلة
+- [ ] قم بإنشاء منسق brainfu * k
+- [ ] أضف تمييز بناء الجملة في REPL
+- [ ] أضف الإكمال التلقائي في REPL
+- [ ] دعم ميزة حجم الصفيف الديناميكي
+
+## مصادر
+- [مخنث فى ويكيبيديا](https://en.wikipedia.org/wiki/Brainfuck)
```
Comment (0x61nas) [inline README-ar.md]: Please don't translate the language name

**Review by 0x61nas — CHANGES_REQUESTED** [review]
Good contribute I like it 💙, But there is only one small adjustment you have to make

File: README-ar.md (hunk 6 above)
Comment (idrismusa4) [reply README-ar.md]: Do you mean the link title?

Comment (0x61nas) [reply README-ar.md]: yep

Comment (idrismusa4) [reply README-ar.md]: Okay will do!

**Review by idrismusa4 — COMMENTED** [review]
Please @anas-elgarhy can you review this and give feedback for any changes if there's any?

**Review by 0x61nas — APPROVED** [review]
Nice
شكرا 💙

**Comment by idrismusa4** [comment]
Thanks so much @anas-elgarhy

**Pull request merged**